java/integTest/src/test/java/glide/standalone/TransactionTests.java

```diff
@@ -121,4 +121,16 @@ public void lastsave() {
         var response = client.exec(new Transaction().lastsave()).get();
         assertTrue(Instant.ofEpochSecond((long) response[0]).isAfter(yesterday));
     }
+
+    @Test
+    @SneakyThrows
+    public void objectIdletime() {
+        String objectIdletimeKey = "key";
+        Transaction transaction = new Transaction();
+        transaction.set(objectIdletimeKey, "");
+        transaction.objectIdletime(objectIdletimeKey);
+        var response = client.exec(transaction).get();
+        assertEquals(OK, response[0]);
+        assertTrue((long) response[1] >= 0L);
```

Comment:
why can't you include this in our regular transaction tests...?

Reply:
Because we don't know the exact value, and can't predict it

Reply:
oh right - transaction tests have to have exact responses.  👍